examples/simple-game-server/Makefile

```diff
@@ -0,0 +1,46 @@
+# Copyright 2017 Google LLC All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+#
+# Makefile for building a simple game-server
+#
+
+#  __     __         _       _     _
+#  \ \   / /_ _ _ __(_) __ _| |__ | | ___ ___
+#   \ \ / / _` | '__| |/ _` | '_ \| |/ _ \ __|
+#    \ V / (_| | |  | | (_| | |_) | |  __\__ \
+#     \_/ \__,_|_|  |_|\__,_|_.__/|_|\___|___/
+#
+
+REPOSITORY = gcr.io/agones-images
+
+mkfile_path := $(abspath $(lastword $(MAKEFILE_LIST)))
+project_path := $(dir $(mkfile_path))
+server_tag = $(REPOSITORY)/game-server:0.1
```

Comment:
Can you call the image `simple-game-server` instead of `game-server` to be consistent with the directory name?

Reply:
So we rather want it to be inconsistent with the other image names?

The udp server image is called `udp-server`, while the directory is called `simple-udp`, so I just want to make sure.

Reply:
Well, @markmandel set the original naming scheme (which I copied for the simple-tcp server) but personally I think it makes more sense if the names match. 

As another data point, the language specific examples are `directory-name`-`server`, e.g. `rust-simple-server` in the directory `rust-simple`.

Reply:
+1 to @roberthbailey makes sense to me for the directory and the name to match :+1:

Reply:
Alright, will rename the image name 👍